blackduck-artifactory-common/src/main/java/com/synopsys/integration/blackduck/artifactory/configuration/ConfigurationPropertyManager.java

```diff
@@ -39,19 +42,25 @@ public ConfigurationPropertyManager(Properties properties) {
     }
 
     public List<String> getRepositoryKeysFromProperties(ConfigurationProperty repositoryKeyListProperty, ConfigurationProperty repositoryKeyCsvProperty) throws IOException {
-        List<String> repositoryKeys;
+        List<String> repositoryKeys = new LinkedList<>();
```

Comment:
What is the purpose of initializing repositoryKeys as a LinkedList here? [Collectors::toList](https://docs.oracle.com/javase/8/docs/api/java/util/stream/Collectors.html#toList--) won't respect this initial assignment. If you want to make sure that it gets collected as a LinkedList you can use [Collectors::toCollection](https://docs.oracle.com/javase/8/docs/api/java/util/stream/Collectors.html#toCollection-java.util.function.Supplier-)

Reply:
I wanted to initialize it to something so that I can check if its empty rather than doing a null check. LinkedList vs ArrayList doesn't really matter in this case, but I believe the team prefers linked list. I'll change it to  `Collectors::toCollection`. Didn't know that existed 👍